modules/health_quest/spec/serializers/v0/va_appointments_serializer_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe HealthQuest::V0::VAAppointmentsSerializer do
+  subject { described_class.new(appt[:data], meta: appt[:meta]) }
+
+  let(:klass) { described_class }
+  let(:user) { build(:user, :health_quest) }
+  let(:appt_service) { HealthQuest::AppointmentService.new(user) }
+  let(:appt) { appt_service.get_appointment_by_id(132) }
+
+  describe 'subject type' do
+    it 'is a HealthQuest::V0::VAAppointmentsSerializer' do
+      expect(subject).to be_a(HealthQuest::V0::VAAppointmentsSerializer)
+    end
+  end
+
+  describe 'included' do
+    it 'includes FastJsonapi::ObjectSerializer' do
+      expect(klass).to include(FastJsonapi::ObjectSerializer)
+    end
+  end
+
+  describe 'KEY #meta' do
+    it 'has meta key' do
+      expect(subject.serializable_hash.key?(:meta)).to be(true)
+    end
+
+    it 'has default values' do
+      expect(subject.serializable_hash[:meta]).to eq({ 'pagination': {
+                                                       'current_page': 0,
+                                                       'per_page': 0,
+                                                       'total_pages': 0,
+                                                       'total_entries': 0
+                                                     } })
+    end
+  end
+
+  describe 'KEY #data' do
+    it 'has data key' do
+      expect(subject.serializable_hash.key?(:data)).to be(true)
+    end
+
+    it 'data has attributes' do
+      expect(subject.serializable_hash[:data][:attributes].keys).to eq(%i[
```

Comment:
Nit: It would look better if you started the `%i[]` on a new line. You wouldn't have to indent all elements 20 times.

Reply:
Sure. Rubocop is still breaking it down into two lines but I think it looks better than the original version.